src/tool/subcommands/api_cmd/api_compare_tests.rs

```diff
@@ -1298,6 +1301,135 @@ fn eth_tests() -> Vec<RpcTest> {
     tests
 }
 
+fn eth_call_api_err_tests(use_alias: bool) -> Vec<RpcTest> {
+    let test_cases = vec![
```

Comment:
Can you find a way to make this leaner? There's a lot of repeated code.

Reply:
Done.